cypress/fixtures/v1/comments/__story-list-comments.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Does Cypress auto-generate these fixtures?

Reply:
nope. you gotta copy/paste